lib/cli/kit.rb

```diff
@@ -46,7 +44,7 @@ class << self
     #   * Think carefully about whether Abort or Bug is more appropriate. Is this
     #       a bug in the tool? Or is it just user error, transient network
     #       failure, etc.?
-    #   * One case where it's ok to rescue these outside of EntryPoint (or tests):
+    #   * One case where it's ok to rescue (other than EntryPoint or tests):
```

Comment:
At the moment, `CLI::Kit` knows nothing of `EntryPoint`, though I think it should provide one.